pwnlib/context/__init__.py

```diff
@@ -335,7 +340,7 @@ class ContextType(object):
     # Setting any properties on a ContextType object will throw an
     # exception.
     #
-    __slots__ = '_tls',
+    __slots__ = ('_tls', )
```

Comment:
```suggestion
    __slots__ = '_tls',
```
There is no need to change it.